### PR TITLE
Added GitBook files to master repository (without mysql) + Appstore documentation

### PR DESCRIPTION
**Pull request by Mark-de-Haan**
#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] Test plan template updated
- [ ] Clean commits

### CODE REVIEW

File: docs/user_documentation/guide-appstore.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+![Activate](../images/appstore/activate_deactivate_apps.png?raw=true, "activate / deactivate")
+
+
+When an app is active, click on the icon to go to your previously created HTML landing page template:
+
+![Landing page](../images/appstore/landing_page.png?raw=true, "landing page")
+
+## Purpose of the App store
+The appstore allows you to create your own webapplications, and place them inside your local MOLGENIS.
+If you are not satisfied with plugins that MOLGENIS offers you, you can write your JS/HTML to display the data in such a way that it fits your needs.
```
Comment (fdlk) [inline docs/user_documentation/guide-appstore.md]: "not satisfied" -> formulation is a bit negative I think, the point is that you can write custom apps to go with your custom data model

File: docs/user_documentation/guide-appstore.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+![Activate](../images/appstore/activate_deactivate_apps.png?raw=true, "activate / deactivate")
+
+
+When an app is active, click on the icon to go to your previously created HTML landing page template:
+
+![Landing page](../images/appstore/landing_page.png?raw=true, "landing page")
+
+## Purpose of the App store
+The appstore allows you to create your own webapplications, and place them inside your local MOLGENIS.
```
Comment (fdlk) [inline docs/user_documentation/guide-appstore.md]: Only a local MOLGENIS?

File: docs/user_documentation/guide-appstore.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+If you are not satisfied with plugins that MOLGENIS offers you, you can write your JS/HTML to display the data in such a way that it fits your needs.
+
+Using REST api calls, you can access data that was previously imported, and use fancy JS magic to create the ultimate data visualisation apps
+
+## Caution
+Apps are stored in the database. If you do not properly version or backup your own apps, then a server crash or a database drop **WILL** delete your app.
+Be smart, backup your apps.
```
Comment (fdlk) [inline docs/user_documentation/guide-appstore.md]: Maybe advise how best to do this?

File: README.md [hunk 4]
```diff
@@ -7,20 +7,22 @@ MOLGENIS is a collaborative open source project on a mission to generate great s
 See http://molgenis.github.io for documentation.
 
 ## Installation
-See [Getting Started Guide](http://molgenis.github.io/develop/start).
+See [Getting Started Guide](docs/develop/start.md).
```
Comment (fdlk) [inline README.md]: Not sure how, but ideally the github README.md links to the gitbook site but the gitbook site itself need not link to itself.

File: docs/developer_documentation/intellij.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+* Select your favorite browser in the pulldown.
+* Save the configuration
+* In the tool bar, select the `molgenis-app [exploded]` configuration and press the play button.
+
+This'll build and deploy molgenis to tomcat and open it in the browser.
+Whenever you tab from molgenis to the browser, all modified resources will be copied to the deployed exploded war.
+A browser reload should display the changes.
+
+## Security
+See also the [MOLGENIS Security settings](./security)
```
Comment (fdlk) [inline docs/developer_documentation/intellij.md]: Relative links don't work either for me.

**Comment by fdlk** [comment]
https://github.com/molgenis/molgenis.org/issues/61 needs to be moved to this repo now I think?

File: README.md (hunk 4 above)
Comment (Mark-de-Haan) [inline README.md]: If I look at how others do this, they actually just link to the MD file on GitHub, and not to the GitBook.

Don't how we want to do this. Easy fix is https://gitbook.com/@molgenis/guide-quickstart.md or something like this

File: docs/developer_documentation/intellij.md (hunk 5 above)
Comment (Mark-de-Haan) [inline docs/developer_documentation/intellij.md]: Yeah it should have the .md extension

**Pull request merged**